extensions/wikia/Forum/tests/ForumTest.php

```diff
@@ -54,17 +54,8 @@ private function getTitleMock( $key ) {
 		return $titleMock;
 	}
 
-	/**
-	 * @param $title Title
-	 * @return PHPUnit_Framework_MockObject_MockObject
-	 */
-	private function getBoardMock( Title $title ) {
-		$boardMock = $this->getMock( 'ForumBoard', [ 'getBoardInfo' ] );
-		$forumBoardInfo = new ForumBoardInfo();
-		$forumBoardInfo->setId( $title->getArticleID() );
-		$boardMock->expects( $this->any() )
-			->method( 'getBoardInfo' )
-			->willReturn( $forumBoardInfo );
+	private function getBoardMock() {
+		$boardMock = $this->getMock( 'ForumBoard', [ 'getBoardInfo', 'getDescriptionWithoutTemplates', 'getTitle' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$boardMock". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)